apps/web-mzima-client/src/app/core/guards/access-allow.guard.ts

```diff
@@ -8,14 +8,13 @@ import { SessionService } from '@services';
 export class AccessAllowGuard implements CanActivate {
   constructor(private router: Router, private service: SessionService) {}
 
-  canActivate(): boolean {
+  canActivate() {
     const access: boolean = this.service.accessToSite;
 
-    if (access) {
-      this.router.navigate([`/map`]);
-    } else {
+    if (!access) {
       return true;
     }
```

Comment:
I think I get confused with the naming here, why is it returning true when there is !access? Is this correct? It works so its probably correct, just curious :)